sbp_generate/sbp.yaml

```diff
@@ -139,8 +139,8 @@
                   desc: Fix mode
                   values:
                       - 0: Single Point Positioning (SPP)
-                      - 1: RTK
-
+                      - 1: RTK fixed 
+                      - 2: RTK float
```

Comment:
Leaving a new line between each message definition makes for easier reading IMO.